winterfell/controllers/fileClaimController.js

```diff
@@ -40,71 +41,68 @@ module.exports = function (app) {
     });
   });
 
-  app.post('/claims/create', function (req, res) {
+  app.post('/claims/create', auth.authenticatedOr404, function (req, res) {
     console.log('[createClaim] request received for ' + JSON.stringify(req.body));
-    var extUserId = req.body.extUserId;
     var callback = function (err, claim) {
       if (claim) {
-        res.status(http.OK).send({claim: FileClaim});
+        res.status(http.CREATED).send(claim);
```

Comment:
Ok to the convention is we return an object which is of the form:

`{ <model_name>: <model_object> }`

that sounds reasonable to me